src/main/kotlin/me/zeroeightsix/kami/feature/module/Aura.kt

```diff
@@ -120,11 +125,17 @@ object Aura : Module() {
         return false
     }
 
+    private fun checkWeapon(stack: ItemStack): Boolean {
+        return (stack.item is AxeItem || stack.item is SwordItem)
```

Comment:
you should add a configurable item value instead of hard coding this. you could do it semilar to [this](https://github.com/zeroeightysix/KAMI/blob/c43c547a286b24dbaed60f7dceb51776fe078d92/src/main/kotlin/me/zeroeightsix/kami/feature/module/ItemHighlight.kt#L18)